misc/config.json

```diff
@@ -11,7 +11,7 @@
         "batch-size": 256,
         "queue-size": 32,
         "yield-delay": 0.5,
-        "long-delay": 120
+        "long-delay": 3600
```

Comment:
Is this commit related to the schema changes?

(also, nitpick, but looking at it now I don't love the name. My suggestion would be `channel-exhaustion-wait` or something at least related to its use)

Reply:
It's not related, this branch is actually called `misc-fixes`, but I titled the PR since that's the main thing that changed.

I'll rename it to something more appropriate.